### DISCUSSION TITLE
I want to close this little thing, what should I do?

### DISCUSSION THREAD

**Discussion by WHBLeer** [Eugeny/tabby#7087] [Q&A] [answered]
<img width="322" alt="image" src="https://user-images.githubusercontent.com/30891605/188214131-3349d1b5-55a9-4feb-9843-88fe27578b3c.png">

**Comment by Eugeny** [comment, marked as answer]
Uninstall the tabby-clippy plugin.

**Reply by WHBLeer** [reply]
Thank you
